client/components/Editor/plugins/reactive/cells/promise.js

```diff
@@ -0,0 +1,25 @@
+const compareArrayFingerprints = (from, to) => {
```

Comment:
We don't use this anywhere yet, and it will probably change or disappear, but it provides some more flavor for what a reactive cell looks like.